cli/src/main/scala/filodb.cli/CliMain.scala

```diff
@@ -25,41 +23,50 @@ import filodb.memory.format.{BinaryVector, Classes, MemoryReader, RowReader}
 import filodb.prometheus.ast.{InMemoryParam, TimeRangeParams, TimeStepParams, WriteBuffersParam}
 import filodb.prometheus.parse.Parser
 import filodb.query._
+import org.rogach.scallop.exceptions.ScallopException
 
 // scalastyle:off
 class Arguments(args: Seq[String]) extends ScallopConf(args) {
+
+
   val dataset = opt[String]()
   val database = opt[String]()
   val command = opt[String]()
   val filename = opt[String]()
-  val configPath = opt[String]()
+  val configpath = opt[String]()
   // max # of results returned. Don't make it too high.
   val limit = opt[Int](default = Some(200))
-  val sampleLimit = opt[Int](default = Some(1000000))
-  val timeoutSeconds = opt[Int](default = Some(60))
+  val samplelimit = opt[Int](default = Some(1000000))
+  val timeoutseconds = opt[Int](default = Some(60))
   val outfile = opt[String]()
-  val delimiter = opt[String](default = Some(",")).apply()
-  val indexName = opt[String]()
+  val indexname = opt[String]()
   val host = opt[String]()
   val port = opt[Int](default = Some(2552))
   val promql = opt[String]()
   val schema = opt[String]()
-  val hexPk = opt[String]()
-  val hexVector = opt[String]()
-  val hexChunkInfo = opt[String]()
-  val vectorType = opt[String]()
+  val hexpk = opt[String]()
+  val hexvector = opt[String]()
+  val hexchunkinfo = opt[String]()
+  val vectortype = opt[String]()
   val matcher = opt[String]()
-  val labelNames = opt[List[String]](default = Some(List()))
-  val labelFilter = opt[Map[String, String]](default = Some(Map.empty))
-  val start: Long = System.currentTimeMillis() / 1000 // promql argument is seconds since epoch
-  val end: Long = System.currentTimeMillis() / 1000 // promql argument is seconds since epoch
+  val labelnames = opt[List[String]](default = Some(List()))
+  val labelfilter = opt[Map[String, String]](default = Some(Map.empty))
+  val start = opt[BigInt](default = Some(System.currentTimeMillis()/1000))// promql argument is seconds since epoch
+  val end = opt[BigInt](default = Some(System.currentTimeMillis()/1000))// promql argument is seconds since epoch
   val minutes = opt[String]()
   val step = opt[Long](default = Some(10)) // in seconds
   val chunks = opt[String]()   // select either "memory" or "buffers" chunks only
-  val everyNSeconds = opt[String]()
+  val everynseconds = opt[String]()
   val shards = opt[List[String]]()
   val spread = opt[Int]()
   verify()
+
+  override def onError(e: Throwable): Unit = e match {
```

Comment:
Why? You always throw the exception back out.

Reply:
This is needed for using intersect in the test spec. else the error returned is a different string  for different errors.